### PR TITLE
flint-checker: add support for go1.16

### PR DESCRIPTION
**Pull request by roopakv**
Add support for go1.16

#47627

### CODE REVIEW

File: Formula/flint-checker.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@ class FlintChecker < Formula
 
   def install
     ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "auto"
     (buildpath/"src/github.com/pengwynn").mkpath
     ln_sf buildpath, buildpath/"src/github.com/pengwynn/flint"
     system "go", "build", "-o", bin/"flint"
```
Comment (SMillerDev) [inline Formula/flint-checker.rb:25]:
```suggestion
    system "go", "build", *std_go_args, "-o", bin/"flint"
```

Comment (roopakv) [reply Formula/flint-checker.rb]: curious why we are adding this here?

**Comment by SMillerDev** [comment]
Is the issue reported upstream?

**Comment by roopakv** [comment]
@SMillerDev https://github.com/pengwynn/flint/issues/54

**Review by Bo98 — APPROVED** [review]

**Pull request merged**